djangocms_versioning/admin.py

```diff
@@ -134,12 +135,36 @@ def _get_publish_link(self, obj):
             {'publish_url': publish_url}
         )
 
+    def _get_edit_link(self, obj):
+        """Helper function to get the html link to the edit action
+        """
+        if obj.state == PUBLISHED:
+            pks_for_grouper = obj.versionable.for_grouper(
+                obj.grouper).values_list('pk', flat=True)
+            content_type = ContentType.objects.get_for_model(obj.content)
+            drafts = Version.objects.filter(
+                object_id__in=pks_for_grouper, content_type=content_type,
+                state=DRAFT)
+            if drafts.exists():
+                return ''
+        elif not obj.state == DRAFT:
+            # Don't display the link if it can't be archived
+            return ''
+        edit_url = reverse('admin:{app}_{model}_edit_redirect'.format(
+            app=obj._meta.app_label, model=self.model.__name__.lower(),
```

Comment:
`model=obj._meta.model_name`

Reply:
If I remember correctly that doesn't work because I need the proxy model here, but I'll try to make sure.